tests/skyserve/load_balancer/service.yaml

```diff
@@ -5,6 +5,7 @@ service:
     initial_delay_seconds: 180
   replica_policy:
     min_replicas: 3
+  load_balancer_type: python
```

Comment:
do we really need this? are we defaulting to plb now?

Reply:
felt like reasonable future proofing, but not a big deal.  Removed it